apollo/plugin.py

```diff
@@ -0,0 +1,173 @@
+from datetime import datetime
+from dataclasses import dataclass
+import httpx
+from typing import Optional, List
+
+from lutraai.augmented_request_client import AugmentedTransport
+
+
+@dataclass
+class EmploymentHistory:
+    organization_name: Optional[str]
+    start_date: Optional[datetime]
+    end_date: Optional[datetime]
+    title: Optional[str]
```

Comment:
i remember type checking/codegen have issues with Optional return values.

may be Opus is better? @vrv what do you think? should we try to reduce # optionals here still?

Reply:
I think we can try to go with this, models should be better.